lib/LedgerSMB/PSGI.pm

```diff
@@ -107,9 +107,6 @@ sub old_app {
                 catch ($e) {
                     exit;
                 }
-                finally {
-                    exit;
-                }
```

Comment:
actually, that's not equivalent.

```plain
try {
...;
}
finally {
   exit;
}
```

can be replaced with:

```plain
try {
   ...;
}
catch ($e) {
}
exit;
```

because finally happens in both the error and the non-error cases.

Reply:
Fixed